nasa-veda-platform/dashboard/index.qmd

```diff
@@ -1,75 +1,39 @@
 ---
 title: VEDA Dashboard
-subtitle: 10 Things to Know About the VEDA Dashboard
+subtitle: Science presentation and entrypoint
 aliases:
   - /services/dashboard.html
 ---
 
-The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. 
+## Goals
 
-Learn more about the principles the VEDA Dashboard is designed around below.
+The VEDA Dashboard is a modular, flexible white-label web application that enable scientific programs and their contributors to visualize, analyze, and communicate scientific data and results.
 
-If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit the [VEDA UI docs](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/development/SETUP.md).
-
-### 1. Science teams need a way to show their work
-- Science teams often need a web presence to show their research program, findings, and methods and data to their stakeholders, colleagues, and other audiences
-- Modern data-driven science is not only presented through papers, posters, and lectures to fellow scientists and through reports to decision makers, but also through up-to-date interactive online tools and blogs that are available directly to either audience.
-Web and cloud platforms can enable scientists to publish contents in these interactive ways.
-- Often, science programs will want to put their information into a specific context and make it discoverable via their name, which is also important for credibility and attribution.
-- VEDA UI helps science teams combine the two - program-specific websites with interactive data visualization and standardized access.
+The ReactJS library offers powerful abstractions for defining and describing data sources (datasets), visualizing them, and exposing them throughout the application in interactive storytelling and data exploration tools.
```

Comment:
This is a bit jargon-ey, I am hoping to avoid this for a top level description of the VEDA Dashboard, wherever possible. I think it's essentially covered by the first two sections of 10 things to know about VEDA Dashboard